avocado/core/test.py

```diff
@@ -1073,15 +1073,16 @@ def cancel(self, message=None):
         """
         raise exceptions.TestCancel(message)
 
-    def fetch_asset(self, name, asset_hash=None, algorithm='sha1',
+    def fetch_asset(self, name, asset_hash=None, algorithm=None,
```

Comment:
I always tend to prefer `None` as defaults. It avoids the need to check the default and simply using `None` :smile: